Cargo.toml

```diff
@@ -23,12 +23,11 @@ lto = "thin"
 codegen-units = 32
 
 [workspace.dependencies]
-tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "cbaff7f180dfa0e5c5c962bc734ef0e47e99290f", features = [
+tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "d837c284bdb37d5806604d6b02374354a821749c", features = [
   "quickwit",        # for sstable support
   "stopwords",
   "lz4-compression",
 ], default-features = false }
-tantivy-common = { git = "https://github.com/paradedb/tantivy.git", rev = "cbaff7f180dfa0e5c5c962bc734ef0e47e99290f" }
```

Comment:
fyi, I realized this dependency isn't actually used, so we don't need to reference it anymore.